dcicutils/beanstalk_utils.py

```diff
@@ -246,7 +246,7 @@ def get_beanstalk_real_url(env):
     if env in urls:
         return urls[env]
 
-    if 'webprod' in env:
+    if 'webprod' in env or 'blue' in env or 'green' in env:
```

Comment:
This code is VERY fragile. Later we should change the environments to be `fourfront-prod-blue` and `fourfront-prod-green` so that blue/green can be used for other things (e.g., `fourfront-mastertest-blue` and `fourfront-mastertest-green`. This is OK for now, but I made a ticket to address it. Please add a comment saying something like:
```
#TODO (C4-91): reconsider environment names
```